tirtha_bk/static/js/main.js

```diff
@@ -7,48 +7,50 @@ const modelArea = doc.getElementById('model-area');
 const model = doc.getElementById('model');
 const fControls = doc.getElementById('floating-controls');
 const infoBtn = doc.getElementById('info-btn');
-const PRE_URL = "" // TODO: Use env
-
+const PRE_URL = ""; // TODO: Use env
+console.log("Is this the file")
 // ========================== FS START ==========================
-// ❗ Handle fullscreen❗
-const expandBtn = doc.getElementById('expand');
-const expParent = expandBtn.parentElement;
+//// ❗ Handle fullscreen❗
+
+// Handle fullscreen
+const expandBtn = $('#expand');
+const expParent = expandBtn.parent();
 
 function isInFullScreen() {
-    return (doc.fullScreenElement && doc.fullScreenElement !== null) || (doc.mozFullScreen || doc.webkitIsFullScreen);
+    return (document.fullScreenElement && document.fullScreenElement !== null) || (document.mozFullScreen || document.webkitIsFullScreen);
 }
 
 function requestFullScreen() {
-    el = modelArea
+    el = modelArea;
     var requestMethod = el.requestFullScreen || el.webkitRequestFullScreen || el.mozRequestFullScreen || el.msRequestFullScreen;
     requestMethod.call(el);
 
     setTimeout(() => {
         model.classList.add("model-fs");
-        doc.querySelector(".controls").style.display = "none";
+        $('.controls').hide();
     }, 100);
 }
 
 function exitFullScreen() {
-    el = doc;
+    el = document;
     var requestMethod = el.cancelFullScreen || el.webkitCancelFullScreen || el.mozCancelFullScreen || el.exitFullscreen || el.webkitExitFullscreen;
     requestMethod.call(el);
 
     setTimeout(() => {
         model.classList.remove("model-fs");
-        doc.querySelector(".controls").style.display = "flex";
+        $('.controls').show();
```

Comment:
`doc.querySelector(".controls").style.display = "flex";` is not the same as `.show()`. Use `$('.controls').css('display', 'flex');`